src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -4,8 +4,9 @@
 #
 
 function CheckIfDurableFunctionsEnabled {
-	if (-not [bool]$env:PSWorkerEnableExperimentalDurableFunctions) {
-		throw 'Durable function is not yet supported for PowerShell.'
+    if (($null -ne $env:PSWorkerEnableExperimentalDurableFunctions) -and
+            (-not [bool]::Parse($env:PSWorkerEnableExperimentalDurableFunctions))) {
+		throw 'PowerShell Durable Functions are disabled (check the PSWorkerEnableExperimentalDurableFunctions environment variable)Durable PowerShell Functions are disabled (check the PSWorkerEnableExperimentalDurableFunctions environment variable).'
```

Comment:
I think this error message should be `PowerShell Durable Functions are disabled (check the PSWorkerEnableExperimentalDurableFunctions environment variable)`

Reply:
Oops, pasted twice by mistake, good catch!